packages/core/src/anchor/ethereum/ethereum-anchor-service.ts

```diff
@@ -150,7 +150,7 @@ export default class EthereumAnchorService implements AnchorService {
       fetch(this.requestsApiEndpoint, {
         method: "POST",
         body: JSON.stringify({
-          docId: cidDocPair.docId.toString(),
+          streamId: cidDocPair.streamId.toString(),
```

Comment:
Can we hold off on updating the request/response to/from the CAS right now?  Otherwise we need to update and deploy the CAS this sprint as well and I'm worried about how quickly the tail end of this project is growing.  We should be able to safely update the CAS apis after ELP launch right?

Reply:
For sure, and good catch. I didn't think about this!